### ISSUE TITLE
archeal genomes 

### ISSUE THREAD

**Issue by Thexiyang** [oschwengers/bakta#48]
Does this tool also work for archeal genomes or metagenome-assembled genomes? Thanks!

**Comment by oschwengers** [comment]
Actually, Bakta was designed & developed to annotate single bacterial genomes. Particularly, its database only comprises bacterial protein coding genes. Also, there are currently no options/parameters to surpass archaeal taxonomic information to external tools within the workflow, _e.g._ tRNAScan-SE.

Regarding metagenomes:
For sure, a very interesting question. I haven't tried it yet but I'll put it on the list and look into this.

**Comment by oschwengers** [comment]
A quick follow-up:
Yes, Bakta works absolutely fine on *bacterial* metagenome-assembled genomes. Actually, due to its large and taxonomically untargeted database, it has a competitive edge compared to Prokka.

**Comment by oschwengers** [comment]
Hey @richardstoeckl, thanks for this follow-up question which gets asked more and more often. I totally see the point in this - I'm just haven't come up with a decision whether it's better to either add an archaea mode to Bakta or to start a new distinct project, maybe archa?

A distinct project would be highly-inspired by Bakta itself sharing a lot of code but would provide room for important distinctions we might have to make, both now and in the future. However, this would induce a lot of code duplication which could only be handled by starting a 3rd library project.

In conclusion, yes there is definitely a chance for a "bakta-like" archaea annotation solution - I'm just not yet convinced what might be the way how to do it. For now, I rather tend to the 2nd idea of a distinct project.

**Comment by GeoMicroSoares** [comment]
Hey @oschwengers I think we've met at VAAM 2024 in Würzburg and chatted a bit about this but just to reinforce - a bakta for archaea - archa as you mentioned - would still be a great tool to have around! Again, great work with bakta & all the best!
